src/network/networkd-manager.c

```diff
@@ -201,7 +201,7 @@ static int manager_udev_process_link(sd_device_monitor *monitor, sd_device *devi
                 return 0;
         }
 
-        if (!STR_IN_SET(action, "add", "change")) {
+        if (!streq(action, "remove")) {
```

Comment:
Ah, sorry, it looks like the debug line should stay for consistency.

`!STR_IN_SET(action, "add", "change")` is not too bad here.  I think in normal operation we only care about "add".

"change" would let us catch a manual `udevadm trigger`, just in case we lost an event (!).  I think that's the potentially confusing part.

E.g. we're not doing anything relevant to "move" events, because we don't use the name here, only the ifindex.

P.S. Why do we think the netlink event (ifindex) is guaranteed to arrive before the udev one?  Should I open a code analysis bug for that?